core/libs/commonwealth/commonwealth/utils/general.py

```diff
@@ -37,8 +37,14 @@ def delete_everything(path: Path) -> None:
 
 
 def file_is_open(path: Path) -> bool:
-    result = subprocess.run(["lsof", path.resolve()], stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=False)
-    return result.returncode == 0
+    try:
+        result = subprocess.run(
+            ["lsof", path.resolve()], stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=False, timeout=5
+        )
+        return result.returncode == 0
+    except Exception as error:
+        logger.error(f"Failed to check if file {path} is open, {error}")
+        return True
```

Comment:
If timed out, the file is considered open.

just commenting so we think about the behavior we are choosing